templates/rsyslog.d/17-docker-journald-logs.shipping.conf.j2

```diff
@@ -0,0 +1,67 @@
+##
+# {{ ansible_managed }}
+#
+
+# Pull docker container log entries from journald and forwards to logstash
+# along with container metadata (container_name, container_id, tag etc.)
+
+module(load="imjournal" StateFile="imjournal.state")
+
+template(
+  name="jsonJournaldLogTemplate"
+  type="list"
+  option.casesensitive="on") {
+    constant(value="{")
+      constant(value="\"@timestamp\":\"")               property(name="timereported" dateFormat="rfc3339" date.inUTC="on")
+      constant(value="\",\"hostname\":\"")              property(name="hostname")
+      constant(value="\",\"container_id\":\"")          property(name="$!CONTAINER_ID")
+      constant(value="\",\"container_name\":\"")        property(name="$!CONTAINER_NAME")
+      constant(value="\",\"container_tag\":\"")         property(name="$!CONTAINER_TAG")
+      constant(value="\",\"role\":\"")                  property(name="$!ROLE")
+      constant(value="\",\"ecs_task_container\":\"")    property(name="$!ECS_TASK_CONTAINER")
```

Comment:
Since this is ECS specific should we reflect this in the name of this file and the switch to enable it? Either that or make these extra meta fields configurable in some way. Also can you add some detail to the README about this new log shipper?

Reply:
Looks good, just needs the README update.